test/bids_examples_ds005.jsonld

```diff
@@ -13,8 +13,7 @@
         },
         "value": 28
       },
-      "internalIdentifier": "Studied state sub-01",
-      "lookupLabel": "Studied state sub-01"
+      "lookupLabel": "sub-01_ses-01"
```

Comment:
this is not what was discussed prior. As reminder please review issue again for bids2ebrains: https://gitlab.ebrains.eu/ri/projects-and-initiatives/bids2ebrains/bids2ebrains/-/issues/12